hack/addon.yaml

```diff
@@ -0,0 +1,9 @@
+---
+kind: Addon
+apiVersion: tackle.konveyor.io/v1alpha1
+metadata:
+  name: tcafeb
```

Comment:
since we don't have human readable name/descriptions for addons. I think it would be wise to consider a more readable one here.